students/BrianNisonger/session4/file_excercise.py

```diff
@@ -0,0 +1,44 @@
+import pathlib
+import collections
+
+def file_1():
+    pth=pathlib.Path("./")
+    for f in pth.iterdir():
+        print(f'{pth.absolute()}\{f}')
+
+def file_2(source,dest):        
+    with open(source, 'rb') as infile, open(dest, 'wb') as outfile:
+        outfile.write(infile.read())
+
+def file_3(source):
+    name_dict={}
+    lang_list=[]
+    with open(source) as f:
+        lines=f.readlines()
+    for line in lines:
+        line_out=line.split(":")
+        if line_out[0] !="Name":
```

Comment:
This is always going to be at the top of the file, so probably better to skip it somehow first, then you don't have to keep checking.